### PR TITLE
fix: allow to create keys with empty hd path

### PR DESCRIPTION
**Pull request by RafilxTenfen**
- Copy the `keys add` command from cosmos-sdk just modified to allow empty HD path (The choice to continue to use the SDK is due the rest of the commands import/export work as expected)
- Move cmd files from package main to eotsd, it helps to debbug

### CODE REVIEW

File: eotsmanager/cmd/eotsd/daemon/keysadd.go [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	cryptokeyring "github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/multisig"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/go-bip39"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
```
Comment (gitferry) [inline eotsmanager/cmd/eotsd/daemon/keysadd.go:29]: maybe we can highlight somewhere that this part of code is adapted from Cosmos SDK and we add a perminent link here?

File: eotsmanager/cmd/eotsd/daemon/keysadd.go [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+			if err != nil {
+				return err
+			}
+
+			if bip39Passphrase != p2 {
+				return errors.New("passphrases don't match")
+			}
+		}
+	}
+
```
Comment (gitferry) [inline eotsmanager/cmd/eotsd/daemon/keysadd.go:278]: So it is where we made the modification, right (removing the hdPath empty check)? Let's highlight what we modified here for future reference

**Review by gitferry — APPROVED** [review]
Great work!

File: eotsmanager/cmd/eotsd/daemon/keysadd.go (hunk 2 above)
Comment (RafilxTenfen) [reply eotsmanager/cmd/eotsd/daemon/keysadd.go:283]: done at https://github.com/babylonlabs-io/finality-provider/pull/148/commits/e03230b89f5ab74fc9eb189df0e9385e9ca41e1f

File: eotsmanager/cmd/eotsd/daemon/keysadd.go (hunk 1 above)
Comment (RafilxTenfen) [reply eotsmanager/cmd/eotsd/daemon/keysadd.go:29]: done at https://github.com/babylonlabs-io/finality-provider/pull/148/commits/e03230b89f5ab74fc9eb189df0e9385e9ca41e1f

**Pull request merged**